modules/gcp/vpc/variables.tf

```diff
@@ -0,0 +1,67 @@
+variable "project_id" {
+  type        = string
+  description = "The ID of the project where this VPC will be created"
+}
+
+variable "region" {
+  type        = string
+  description = "Region in which to deploy the resources"
+}
+
+variable "network_name" {
+  description = "The name of the network being created"
+}
+
+variable "subnets" {
+  type        = list(map(string))
```

Comment:
Define the proper type please for your variables.

Reply:
It shouldn't be `list(map(string))`, it should be `list(object({ ... myvars }))`